### ISSUE TITLE
Make t/follow.t use a temp directory

### ISSUE THREAD

**Issue by petdance** [petdance/file-next#17]
This test failure is (I'm guessing) because of conflicts between builds.

From ether@cpan.org:

```
This distribution has been tested as part of the CPAN Testers
project, supporting the Perl programming language.  See
http://wiki.cpantesters.org/ for more information or email
questions to cpan-testers-discuss@perl.org


--
Dear PETDANCE,

This is a computer-generated report for File-Next-1.12
on perl 5.19.6, created by App::cpanminus::reporter 0.10 (1.7001).

Thank you for uploading your work to CPAN.  However, there was a problem
testing your distribution.

If you think this report is invalid, please consult the CPAN Testers Wiki
for suggestions on how to avoid getting FAIL reports for missing library
or binary dependencies, unsupported operating systems, and so on:

http://wiki.cpantesters.org/wiki/CPANAuthorNotes

Sections of this report:

   * Tester comments
   * Program output
   * Prerequisites
   * Environment and other context

------------------------------
TESTER COMMENTS
------------------------------

Additional comments from tester:

none provided

------------------------------
PROGRAM OUTPUT
------------------------------

Output from '':

Building and testing File-Next-1.12
cp Next.pm blib/lib/File/Next.pm
Manifying blib/man3/File::Next.0
PERL_DL_NONLAZY=1 /Users/ether/perl5/perlbrew/perls/19.6/bin/perl5.19.6 "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'blib/lib', 'blib/arch')" t/*.t
# Testing File::Next 1.12, Perl 5.019006, /Users/ether/perl5/perlbrew/perls/19.6/bin/perl5.19.6
t/00-load.t ....... ok
t/dirs.t .......... ok
t/dot.t ........... ok
t/basic.t ......... ok

#   Failed test 'NO_PARMS'
#   at t/everything.t line 66.
#     Structures begin differing at:
#          $got->[37] = 't/swamp/linkdir'
#     $expected->[37] = 't/swamp/parrot.pir'

#   Failed test 'NO_PARMS'
#   at t/everything.t line 92.
#     Structures begin differing at:
#          $got->[8] = 't/swamp/linkdir/a1'
#     $expected->[8] = 't/zero.t'
# Looks like you failed 2 tests of 4.
t/everything.t .... 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/4 subtests 
t/pod-coverage.t .. skipped: Test::Pod::Coverage 1.04 required for testing POD
t/pod.t ........... skipped: Test::Pod 1.14 required for testing POD
t/api.t ........... ok
t/named-pipe.t .... ok
t/methods.t ....... ok
t/follow.t ........ ok
t/from_stdin.t .... ok
t/parms.t ......... ok
t/from_file.t ..... ok
t/sort.t .......... ok
t/zero.t .......... ok

Test Summary Report
-------------------
t/everything.t  (Wstat: 512 Tests: 4 Failed: 2)
 Failed tests:  2, 4
 Non-zero exit status: 2
Files=16, Tests=82,  0 wallclock secs ( 0.08 usr  0.05 sys +  0.64 cusr  0.17 csys =  0.94 CPU)
Result: FAIL

------------------------------
PREREQUISITES
------------------------------

Prerequisite modules loaded:

   No requirements found

------------------------------
ENVIRONMENT AND OTHER CONTEXT
------------------------------

Environment variables:

   LANG = en_CA.UTF-8
   PATH = /Users/ether/.perlbrew/libs/19.7@std/bin:/Users/ether/perl5/perlbrew/bin:/Users/ether/perl5/perlbrew/perls/19.7/bin:/Users/ether/bin:/Users/ether/git/misc/install/tequila/bin:/Users/ether/perl5/perlbrew/bin:/Users/ether/perl5/perlbrew/perls/19.6/bin:/opt/local/bin:/opt/local/sbin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin:/usr/X11/bin:/opt/local/share/git-core/contrib/workdir
   PERL5LIB = /Users/ether/.perlbrew/libs/19.7@std/lib/perl5
   PERLBREW_BASHRC_VERSION = 0.67
   PERLBREW_HOME = /Users/ether/.perlbrew
   PERLBREW_LIB = std
   PERLBREW_MANPATH = /Users/ether/.perlbrew/libs/19.7@std/man:/Users/ether/perl5/perlbrew/perls/19.7/man
   PERLBREW_PATH = /Users/ether/.perlbrew/libs/19.7@std/bin:/Users/ether/perl5/perlbrew/bin:/Users/ether/perl5/perlbrew/perls/19.7/bin
   PERLBREW_PERL = 19.7
   PERLBREW_ROOT = /Users/ether/perl5/perlbrew
   PERLBREW_VERSION = 0.67
   PERLDOC_PAGER = less -sicMr
   PERL_AUTOINSTALL_PREFER_CPAN = 1
   PERL_LOCAL_LIB_ROOT = /Users/ether/.perlbrew/libs/19.7@std
   PERL_MB_OPT = --install_base /Users/ether/.perlbrew/libs/19.7@std
   PERL_MM_OPT = INSTALL_BASE=/Users/ether/.perlbrew/libs/19.7@std
   SHELL = /bin/bash
   TERM = xterm-color
   TMPDIR = /var/folders/+h/+hzG2W8MHem4qQiRP616jE+++TQ/-Tmp-/

Perl special variables (and OS-specific diagnostics, for MSWin32):

   EGID = 20 20 402 204 100 98 81 80 79 61 12 401 403
   EUID = 503
   EXECUTABLE_Andy Lester = /Users/ether/perl5/perlbrew/perls/19.6/bin/perl5.19.6
   GID = 20 20 402 204 100 98 81 80 79 61 12 401 403
   UID = 503

Perl module toolchain versions installed:

   Module              Have    
   ------------------- --------
   CPAN                2.03    
   CPAN::Meta          2.132830
   Cwd                 3.45    
   ExtUtils::CBuilder  0.280212
   ExtUtils::Command   1.18    
   ExtUtils::Install   1.61    
   ExtUtils::MakeMaker 6.84    
   ExtUtils::Manifest  1.63    
   ExtUtils::ParseXS   3.23    
   File::Spec          3.45    
   JSON                2.90    
   JSON::PP            2.27203 
   Module::Build       0.4203  
   Module::Signature   n/a     
   Parse::CPAN::Meta   1.4409  
   Test::Harness       3.30    
   Test::More          1.001002
   YAML                n/a     
   YAML::Syck          1.27    
   version             0.9904  


--

Summary of my perl5 (revision 5 version 19 subversion 6) configuration:

 Platform:
   osname=darwin, osvers=10.8.0, archname=darwin-2level
   uname='darwin tequila 10.8.0 darwin kernel version 10.8.0: tue jun 7 16:33:36 pdt 2011; root:xnu-1504.15.3~1release_i386 i386 '
   config_args='-de -Dprefix=/Users/ether/perl5/perlbrew/perls/19.6 -Dman1dir=none -Dman3dir=none -Dusedevel -Aeval:scriptdir=/Users/ether/perl5/perlbrew/perls/19.6/bin'
   hint=recommended, useposix=true, d_sigaction=define
   useithreads=undef, usemultiplicity=undef
   useperlio=define, d_sfio=undef, uselargefiles=define, usesocks=undef
   use64bitint=define, use64bitall=define, uselongdouble=undef
   usemymalloc=n, bincompat5005=undef
 Compiler:
   cc='cc', ccflags ='-fno-common -DPERL_DARWIN -fno-strict-aliasing -pipe -fstack-protector -I/usr/local/include -I/opt/local/include',
   optimize='-O3',
   cppflags='-fno-common -DPERL_DARWIN -fno-strict-aliasing -pipe -fstack-protector -I/usr/local/include -I/opt/local/include'
   ccversion='', gccversion='4.2.1 (Apple Inc. build 5666) (dot 3)', gccosandvers=''
   intsize=4, longsize=8, ptrsize=8, doublesize=8, byteorder=12345678
   d_longlong=define, longlongsize=8, d_longdbl=define, longdblsize=16
   ivtype='long', ivsize=8, nvtype='double', nvsize=8, Off_t='off_t', lseeksize=8
   alignbytes=8, prototype=define
 Linker and Libraries:
   ld='env MACOSX_DEPLOYMENT_TARGET=10.3 cc', ldflags =' -fstack-protector -L/usr/local/lib -L/opt/local/lib'
   libpth=/usr/local/lib /opt/local/lib /usr/lib
   libs=-lgdbm -ldbm -ldl -lm -lutil -lc
   perllibs=-ldl -lm -lutil -lc
   libc=, so=dylib, useshrplib=false, libperl=libperl.a
   gnulibc_version=''
 Dynamic Linking:
   dlsrc=dl_dlopen.xs, dlext=bundle, d_dlsymun=undef, ccdlflags=' '
   cccdlflags=' ', lddlflags=' -bundle -undefined dynamic_lookup -L/usr/local/lib -L/opt/local/lib -fstack-protector'


Characteristics of this binary (from libperl): 
 Compile-time options: HAS_TIMES PERLIO_LAYERS PERL_DONT_CREATE_GVSV
                       PERL_HASH_FUNC_ONE_AT_A_TIME_HARD PERL_MALLOC_WRAP
                       PERL_NEW_COPY_ON_WRITE PERL_PRESERVE_IVUV
                       PERL_USE_DEVEL USE_64_BIT_ALL USE_64_BIT_INT
                       USE_LARGE_FILES USE_LOCALE USE_LOCALE_COLLATE
                       USE_LOCALE_CTYPE USE_LOCALE_NUMERIC USE_PERLIO
                       USE_PERL_ATOF
 Built under darwin
 Compiled at Nov 24 2013 14:40:18
 %ENV:
   PERL5LIB="/Users/ether/.perlbrew/libs/19.7@std/lib/perl5"
   PERLBREW_BASHRC_VERSION="0.67"
   PERLBREW_HOME="/Users/ether/.perlbrew"
   PERLBREW_LIB="std"
   PERLBREW_MANPATH="/Users/ether/.perlbrew/libs/19.7@std/man:/Users/ether/perl5/perlbrew/perls/19.7/man"
   PERLBREW_PATH="/Users/ether/.perlbrew/libs/19.7@std/bin:/Users/ether/perl5/perlbrew/bin:/Users/ether/perl5/perlbrew/perls/19.7/bin"
   PERLBREW_PERL="19.7"
   PERLBREW_ROOT="/Users/ether/perl5/perlbrew"
   PERLBREW_VERSION="0.67"
   PERLDOC_PAGER="less -sicMr"
   PERL_AUTOINSTALL_PREFER_CPAN="1"
   PERL_LOCAL_LIB_ROOT="/Users/ether/.perlbrew/libs/19.7@std"
   PERL_MB_OPT="--install_base /Users/ether/.perlbrew/libs/19.7@std"
   PERL_MM_OPT="INSTALL_BASE=/Users/ether/.perlbrew/libs/19.7@std"
 @INC:
   /Users/ether/.perlbrew/libs/19.7@std/lib/perl5/darwin-2level
   /Users/ether/.perlbrew/libs/19.7@std/lib/perl5
   /Users/ether/perl5/perlbrew/perls/19.6/lib/site_perl/5.19.6/darwin-2level
   /Users/ether/perl5/perlbrew/perls/19.6/lib/site_perl/5.19.6
   /Users/ether/perl5/perlbrew/perls/19.6/lib/5.19.6/darwin-2level
   /Users/ether/perl5/perlbrew/perls/19.6/lib/5.19.6
   .
```

**Comment by petdance** [comment]
Anybody have a patch for this handy?

**Comment by petdance** [comment]
Fixed with 0f2d2dc298b4ac339a3b2d198208014c4c814c36 and releasing in 1.13_02.
